nest/src/config.ts

```diff
@@ -0,0 +1,23 @@
+export const config = () => ({
+    census: {
+        serviceID: process.env.CENSUS_SERVICE_ID,
+        environment: 'ps2',
+        worlds: process.env.CENSUS_SUBSCRIPTION_WORLDS.split(',') ?? ['all'],
+    },
+
+    rabbitmq: {
+        urls: [process.env.RMQ_URL ?? 'amqp://localhost:5672'],
+        queue: process.env.RMQ_QUEUE ?? 'aggregator',
+        queueOptions: {
+            durable: false,
+        },
+    },
+
+    api: {
+        urls: [process.env.RMQ_API_URL ?? 'amqp:localhost:5672'],
```

Comment:
This will be the same, there's no plans to have more than 1 MQ server

Reply:
I am aware, we can configure it to be the same.